### PR TITLE
Add trending-words

### PR DESCRIPTION
**Pull request by spiderjako**

### CODE REVIEW

File: san/query.py [hunk 1]
```diff
@@ -3,13 +3,19 @@
...
 
 def get_gql_query(idx, dataset, **kwargs):
-    query, slug = parse_dataset(dataset)
-    return getattr(
-        san.sanbase_graphql,
-        query,
-        lambda *args, **kwargs: not_found(query)
-    )(idx, slug, **kwargs)
-
+    if "/" in dataset:
```
Comment (IvanIvanoff) [inline san/query.py]: What's `dataset` here? It seems to me that it's a string. 
Also, what's the logic behind `if "/" in dataset`?

Comment (spiderjako) [reply san/query.py]: The dataset is a string, containing "metric/slug", so if it doesn't have a "/" there, then there isn't a slug. The problem is, that one of the tests is failing atm and I'm trying to figure that one out

File: san/sanbase_graphql.py [hunk 2]
```diff
@@ -440,6 +440,28 @@ def token_top_transactions(idx, slug, **kwargs):
...
+    query_str = """
+    query_{idx}: getTrendingWords (
+        from: \"{from_date}\",
+        to: \"{to_date}\",
+        size: {size},
+        interval: \"{interval}\"
+    ){{
+        datetime,
+        topWords{{
+            score,
```
Comment (IvanIvanoff) [inline san/sanbase_graphql.py]: omit the commas in the query

File: san/query.py [hunk 3]
```diff
@@ -2,15 +2,21 @@
 from san.error import SanError
 
 
-def get_gql_query(idx, dataset, **kwargs):
-    query, slug = parse_dataset(dataset)
+def get_gql_query(idx, identifier, **kwargs):
+  query, _separator, slug = identifier.partition("/")
+  if slug == '':
```
Comment (IvanIvanoff) [inline san/query.py]: btw, this is not guarding against passing something like `daily_active_addresses/`. Maybe the first case should be `if separator != '' and slug == '': <error>`

File: README.md [hunk 4]
```diff
@@ -1233,6 +1234,36 @@ datetime                      value
 2019-09-03 00:00:00+00:00    9.0
 ```
 
+### Trending Words
```
Comment (IvanIvanoff) [inline README.md]: Let's use the new name `Emerging Trends` instead of `Trending Words` even though the API has the old name

**Pull request merged**